### ISSUE TITLE
[Suggestion] Add Fusion style

### ISSUE THREAD

**Issue by adem4ik** [FelixdelasPozas/TrayWeather#164]
Version: 1.29.0

1. Start TrayWeather from the command line using the following option:
` .\TrayWeather.exe -style fusion`
![изображение](https://github.com/FelixdelasPozas/TrayWeather/assets/4707112/5a7733f2-312e-49e5-bf01-eb0e183ab72d)

Result: It seems like TrayWeather supports Fusion style out of the box, it just needs GUI setting.

![изображение](https://github.com/FelixdelasPozas/TrayWeather/assets/4707112/ed000f21-9b7c-4a86-b151-8aa2935bce16)

**Comment by FelixdelasPozas** [comment]
Yes, the Qt library is compiled with the usual styles for Windows (i just looked, they are "Windows", "WindowsXP", "WindowsVista" & "Fusion"). But I certainly prefer Qt to try to use the style that is "closest" to the OS (I like the progress bars and the sliders, Fusion are more "Windows 7" sliders). 

I could set the style in [main.cpp](https://github.com/FelixdelasPozas/TrayWeather/blob/master/Main.cpp) like this:

>59  QApplication app(argc, argv);
>60  app.setQuitOnLastWindowClosed(false);
**61  app.setStyle(QStyleFactory::create("WindowsXP")); // this one line.**

Qt will always allow the user to set the style in the windows arguments, exactly as you described. I think it will confuse users if I put an option in the "Miscellaneous" tab about "GUI style", as they all are very similar. 

It could be added as a third option in the "Visual Theme" option (the light UI/dark UI option in Miscellaneous tab) as the dark theme is a complete "reskin" of the widgets, more or less the same thing the "Fusion" theme does. 

I'll test if it messes with the UI in some parts. Adding it as a third option costs next to nothing. 

Regards, 
Félix.
